packages/pagination/src/styled/OffsetPagination/StyledList.ts

```diff
@@ -8,13 +8,13 @@
 import styled from 'styled-components';
 import { retrieveComponentStyles, DEFAULT_THEME, getColor } from '@zendeskgarden/react-theming';
 
-const COMPONENT_ID = 'pagination.pagination_view';
+const COMPONENT_ID = 'pagination.list';
 
 /**
  * 1. List reset.
  * 2. Text truncation.
  */
-export const StyledPagination = styled.ul.attrs({
+export const StyledList = styled.ul.attrs({
```

Comment:
Not sure, due to the arrows, but is it more semantically appropriate for this to be a `styled.ol`?

Reply:
The resources I cited in the RFC seemed consistent with `ul` for the list element, so I ran with that. My worry with `ol` would also be that the announcement of the `li` number would create confusion next to the page number. `list item 3, page 1`, that sort of thing.

Reply:
Ah, right. Thanks for doing the thorough research on this.